### PR TITLE
fix: make sprt result format consistent with cutechess output

### PR DESCRIPTION
**Pull request by gahtan-syarif**
looks like this for cutechess output:
```
SPRT: llr 2.98 (101.2%), lbound -2.94, ubound 2.94 - H1 was accepted
```

### CODE REVIEW

File: app/src/matchmaking/output/output_cutechess.hpp [hunk 1]
```diff
@@ -35,8 +35,22 @@ class Cutechess : public IOutput {
...
+            double percentage = llr < 0 ? llr / lowerBound * 100 : llr / upperBound * 100;
+           
+            std::string result;
+            if (llr >= upperBound) {
+                result = " - H1 was accepted";
+            } else if (llr < lowerBound) {
+                result = " - H0 was accepted";
+            } else {
+                result = "";
+            }
```
Comment (Disservin) [inline app/src/matchmaking/output/output_cutechess.hpp:50]: this is useless?

**Review by Disservin — CHANGES_REQUESTED** [review]

File: app/src/matchmaking/output/output_cutechess.hpp [hunk 2]
```diff
@@ -35,8 +35,22 @@ class Cutechess : public IOutput {
...
+           
+            std::string fmt = fmt::format("SPRT: llr {.2f} ({.1f}%), lbound {}, ubound {}{}\n",
```
Comment (Disservin) [inline app/src/matchmaking/output/output_cutechess.hpp:52]: also just return fmt::format directly here

File: app/src/matchmaking/output/output_cutechess.hpp (hunk 1 above)
Comment (gahtan-syarif) [reply app/src/matchmaking/output/output_cutechess.hpp]: removed 👍

File: app/src/matchmaking/output/output_cutechess.hpp (hunk 2 above)
Comment (gahtan-syarif) [reply app/src/matchmaking/output/output_cutechess.hpp]: fixed 👍

**Pull request merged**